package.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "remark-language-server",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```

Start w/ 0, in separate commit, when `npm publish`, then update